pinc/DPage.inc

```diff
@@ -461,13 +461,33 @@ function Page_clearRound(string $projectid, string $image, Round $round, string
 
 // -----------------------------------------------------------------------------
 
+// The numeric values of the following array must not change as they are
+// saved in the database to indicate the bad page reason.
 $PAGE_BADNESS_REASONS = [
-    '',
-    _('Missing Image'),
-    _('Missing Text'),
-    _('Image/Text Mismatch'),
-    _('Corrupted Image'),
-    _('Other'),
+    0 => [
```

Comment:
Maybe remove "none" since it is not a valid reason. And make necessary adjusments elsewhere.

Reply:
Done this.